src/task-actions.ts

```diff
@@ -339,31 +339,44 @@ async function handleContractVerification(
     const paths = [mainPath];
 
     const bodyFormData = new FormData();
-    bodyFormData.append("contract-name", path.parse(mainPath).base);
+    bodyFormData.append("compiler-version", args.compilerVersion);
+    bodyFormData.append("license", args.license || "No License (None)");
+    bodyFormData.append("account-contract", args.accountContract || "false");
 
     // Dependencies (non-main contracts) are in args.paths
     if (args.paths) {
         paths.push(...args.paths);
     }
 
-    handleMultiPartContractVerification(bodyFormData, paths, hre.config.paths.root);
-
-    await axios
-        .post(voyagerUrl, bodyFormData.getBuffer(), {
-            headers: bodyFormData.getHeaders()
-        })
-        .catch(() => {
+    const sourceRegex = new RegExp("^" + hre.config.paths?.starknetSources + "/");
+    const contractNameDefault = mainPath.replace(sourceRegex, "");
```

Comment:
Will this work for files outside of the project? I'm not saying that we should support it, just trying to understand.

Reply:
This will only work for files in the project.